internal/gatewayapi/conformance/suite.go

```diff
@@ -27,9 +28,26 @@ func skipTestsShortNames(skipTests []suite.ConformanceTest) []string {
 
 // EnvoyGatewaySuite is the conformance suite configuration for the Gateway API.
 var EnvoyGatewaySuite = suite.ConformanceOptions{
-	SupportedFeatures: features.AllFeatures,
-	ExemptFeatures: sets.New[features.SupportedFeature]().
-		Insert(features.MeshCoreFeatures.UnsortedList()...).
-		Insert(features.MeshExtendedFeatures.UnsortedList()...),
-	SkipTests: skipTestsShortNames(SkipTests),
+	SupportedFeatures: allFeatures(),
+	ExemptFeatures:    meshFeatures(),
+	SkipTests:         skipTestsShortNames(SkipTests),
+}
+
+func allFeatures() sets.Set[features.FeatureName] {
+	allFeatures := sets.New[features.FeatureName]()
+	for _, feature := range features.AllFeatures.UnsortedList() {
+		allFeatures.Insert(feature.Name)
+	}
+	return allFeatures
+}
+
+func meshFeatures() sets.Set[features.FeatureName] {
+	meshFeatures := sets.New[features.FeatureName]()
+	for _, feature := range features.MeshCoreFeatures.UnsortedList() {
```

Comment:
prefer `Insert(features.MeshCoreFeatures.UnsortedList()...` since its more readable

Reply:
features.Feature was a string, but it's been changed to a structure in Gateway API v1.2.0